orchagent/neighorch.cpp

```diff
@@ -72,6 +72,14 @@ bool NeighOrch::resolveNeighborEntry(const NeighborEntry &entry, const MacAddres
     return true;
 }
 
+void NeighOrch::clearResolvedNeighborEntry(const NeighborEntry &entry)
+{
+    string key, alias = entry.alias;
+    key = alias + ":" + entry.ip_address.to_string();
+    m_appNeighResolveProducer.del(key);
```

Comment:
Do we need to check the key presence and delete?

Reply:
I think this is not necessary since this del would only be triggered when a set of the key happened before. Therefore, the scenario of deleting a non-existing key should never happen. Also, it seems that there is no side effect from an attempt of removing a non-existing key.

Reply:
ok fine.

Reply:
@shi-su m_appNeighResolveProduce table is being for processFDBFlushUpdate as well, hope we don't do nbr resolve for FDB flush and add Nexthop at the same time, please check this.

Reply:
Good point. IMO, it would be a rare scenario in which case it should be fine even if we attempt to resolve twice

Reply:
ok

Reply:
@shi-su Looks like it is possible that doResolveNeighTask might resolve the nbr when we delete the entry from APP_DB, please fix it by checking op == DEL_COMMAND.

Reply:
Thanks for this comment. Skipping neighbor resolution in DEL_COMMAND